2_attributePoints.R

```diff
@@ -31,7 +26,8 @@ shrtNms <- merge(data.frame(fileName = raslist.short, fullname = raslist), evs)
 dbDisconnect(db)
 
 gridlist <- as.list(paste(loc_envVars,shrtNms$fullname,sep = "/"))
-names(gridlist) <- shrtNms$gridName
+nm <- substr(shrtNms$fullname,1,nchar(shrtNms$fullname) - 4) # remove .tif extension
```

Comment:
hmm, this looks wrong to me. Names need to be short, as they are going to column names in attribute table, right?

Reply:
These are only used in this script and wouldn't carry over to the output file; they get renamed in line 139